integration

```diff
@@ -778,6 +778,7 @@
   "RobertD502/home-assistant-petkit",
   "RobHofmann/HomeAssistant-GreeClimateComponent",
   "RobHofmann/HomeAssistant-PhilipsAndroid2014",
+  "robinostlund/homeassistant-svk-mimer",
```

Comment:
<https://github.com/robinostlund/homeassistant-svk-mimer/blob/master/hacs.json>
This should be locked to SE, as this is only for users in that country.

<https://hacs.xyz/docs/publish/start/#hacsjson> (see `country`)

Reply:
That is correct, i have added contry now :)